config/webpack.config.js

```diff
@@ -131,7 +131,7 @@ module.exports = {
         // its runtime that would otherwise processed through "file" loader.
         // Also exclude `html` and `json` extensions so they get processed
         // by webpacks internal loaders.
-        exclude: [/\.(js|jsx|mjs|ts|tsx)$/, /\.css$/, /\.html$/, /\.json$/],
+        exclude: [/\.(js|jsx|mjs|ts|tsx|ejs)$/, /\.css$/, /\.html$/, /\.json$/],
```

Comment:
why?

Reply:
I'm using our webpack config in the storybook. The storybook uses `ejs` on the site and can't be load by the `file-loader`, the explanation for this is in the comments above this code.